src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -366,6 +517,17 @@ export default class MissionWindow extends Component<ThemeProps, State> {
           <h1 style={{ marginTop: 0 }}>Options</h1>
           <MissionOptions title={title} missionNames={missionNames} options={options} />
         </div>
+        <div className="geofenceContainer">
+          <h1> GeoFencing </h1>
+          <input className="inputFields" type="number" name="geofenceScanTop" value={checklist.geofenceTop || ''} disabled={locked.geofence} onChange={this.onChange} placeholder="Top" />
+          <br />
+          <input className="inputFields" type="number" name="geofenceBottom" value={checklist.geofenceBottom || ''} disabled={locked.geofence} onChange={this.onChange} placeholder="Bottom" />
+          <br />
+          <input className="inputFields" type="number" name="geofenceLeft" value={checklist.geofenceLeft || ''} disabled={locked.geofence} onChange={this.onChange} placeholder="Left" />
+          <br />
+          <input className="inputFields" type="number" name="geofenceRight" value={checklist.geofenceRight || ''} disabled={locked.geofence} onChange={this.onChange} placeholder="Right" />
+          <CreateBoundingBoxButton theme={theme} name="geofence" value="Bounding Box" />
```

Comment:
shouldnt be Bounding Box

Reply:
just the value="Bounding Box" right?

Reply:
if i would change every function with the word "boundingbox" would it break the code?

Reply:
so when you have value="Bounding Box" its calling that specific box "Bounding Box", you'd want to change it to "Geofencing" or "Geofencing Box". This is completely different from the name="geofence" field